README.md

```diff
@@ -1,14 +1,30 @@
-CTSM_PY
-=======
+CTSM Python Gallery
+===================
 
-A place to put sample workflows and tools that use ctsm model output
+This repository includes a series of Jupyter Notebooks that highlight how Python can be used to analyze CTSM and CMIP-like model output.
+
+## Notebooks
+
+| Notebook      | Author        | Description  |
+| ------------- |---------------| -------------|
+| AnomalyDetrend.ipynb | Will Wieder | TODO |
+| compare_esm_ssp_opt.ipynb | Katie Dagon | TODO |
+| LandUseChange_maps.ipynb | Dave Lawrennce | TODO |
+| MonthlySVD.ipynb | Will Wieder | TODO |
+| ScatterExample.ipynb | Danica Lombardozzi | TODO |
+| SimpleExample.ipynb | Will Wieder | TODO |
+| WeightedMeans.ipynb | Katie Dagon | TODO |
```

Comment:
@wwieder, @katiedagon, @djk2120, can you provide 1 line descriptions of each of these notebooks.

Reply:
@jhamman what's the easiest way to do this.  I'd like to just get the branch on my fork and directly modify the markdown file on github, or do I have to copy/ clone your form remotely?

Reply:
Never mind, edits can be made directly on Joe's branch, as i'd hoped...

Reply:
I edited the README.md to add my descriptions on @jhamman gallery branch and created a pull request. It doesn't show up here yet but I'm hoping I did it right.